hip/get_info.cmake

```diff
@@ -1,12 +1,10 @@
 ginkgo_print_module_header(${detailed_log} "HIP")
-ginkgo_print_foreach_variable(${detailed_log}
-    "GINKGO_HIPCONFIG_PATH")
```

Comment:
I still think you can remove `GINKGO_HIPCONFIG_PATH` fully. It's not used, only set.

Reply:
oh sorry, I skipped over that part of the comment